schemas/+aod/ScanMotion.m

```diff
@@ -27,16 +27,24 @@ function makeTuples(self, key)
 		
             tuple = key;
             
-            br = getFile(acq.AodScan & key,'Motion');
-            motionData = br(:,:,:);
-            [T D P] = size(motionData);
-            mot = reshape(motionData,[T 10 20 P]);
-            mot = permute(mot, [4 2 3 1]);
-            t = br(:,'t');
-            coordinates = br.motionCoordinates;
-            
+            fn = char(fetchn(acq.AodScan & key, 'aod_scan_filename')) ; % get the filename from db
+            oldversion = aod.checkFileversion(fn) ; % check if motion data is in old format, i.e. 2 planes or in new format, i.e. a volume

```

Comment:
this is a typo, or more likely the checkFileVersion should have been checked into the aod directory instead of the acq directory.